src/renderer/components/ScreenSharePicker.tsx

```diff
@@ -83,11 +83,14 @@ addPatch({
 
 if (isLinux) {
     onceReady.then(() => {
-        FluxDispatcher.subscribe("VOICE_STATE_UPDATES", e => {
-            for (const state of e.voiceStates) {
-                if (state.userId === UserStore.getCurrentUser().id && state.oldChannelId && !state.channelId)
-                    VesktopNative.virtmic.stop();
+        FluxDispatcher.subscribe("STREAM_CLOSE", ({ streamKey }: { streamKey: string }) => {
```

Comment:
does this only handle streams or could this potentially also be the camera? iirc you can both share camera and screen at once

Reply:
> does this only handle streams or could this potentially also be the camera? iirc you can both share camera and screen at once

Good catch! Will investigate

Reply:
Can confirm that this is only sent by Streams, not Cameras